target/ppc/cpu.h

```diff
@@ -2672,4 +2677,21 @@ void dump_mmu(CPUPPCState *env);
 void ppc_maybe_bswap_register(CPUPPCState *env, uint8_t *mem_buf, int len);
 void ppc_store_vscr(CPUPPCState *env, uint32_t vscr);
 uint32_t ppc_get_vscr(CPUPPCState *env);
+
+    /*
+     * Must be moved to a more appropriate file
+     */
+enum GEROp {
+    GER_SAT = 0x8,    /* Saturate */
+    GER_ACC = 0x4,    /* Accumulate*/
+    GER_NM  = 0x2,    /* Negative multiply */
+    GER_NA  = 0x1,    /* Negative accumulate */
+    /* Combinations of the above. */
+    GER_NOP = 0,
+    GER_PP  = GER_ACC,
+    GER_SPP = GER_SAT | GER_ACC,
+    GER_NP  = GER_ACC | GER_NM,
+    GER_PN  = GER_ACC | GER_NA,
+    GER_NN  = GER_ACC | GER_NM | GER_NA,
+};
 #endif /* PPC_CPU_H */
```

Comment:
I've put here this enum, but I don't think this is the best place, so does anyone have any suggestions?